content/articles/how-to-build-a-task-manager-with-angular/index.md

```diff
@@ -0,0 +1,468 @@
+# How to build a task manager with angular.
+
+## Introduction
+
+If you are trying to learn a programming language or framework, it is good practice to build simple tasks which will go a long way to help solidify your knowledge in that language.  
+
+In this article, I will work you through building a simple task manager app (popularly known as a Todo app) using [angular](https://angular.io/docs) a JavaScript framework.
+
+
+## What is a Task Manager?
+
+A task manager is used to write, organize and rearrange tasks more efficiently. The task manager we are going to build will allow the user to add a task, mark the task as complete and also delete the task.
```

Comment:
This sentence is hard to follow.. something like `A task manager writes, organizes, and rearranges tasks more efficiently. The task manager we will build will allow the user to add a task, mark the task as complete, and delete the task.` Try making them as simple as possible!